File: Python-164/chapter.4/lab4.15/lab_program.py
# Instructions are imported from java 164 class
# The syntax && logic flow might not follow exact instructions.
'''
/*
A “jiffy” is the scientific name for 1/100th of a second. Given an input number of seconds, output the number of "jiffies."

Output each floating-point value with two digits after the decimal point, which can be achieved as follows:
System.out.printf("%.2f", yourValue);

Ex: If the input is:

15

the output is:

1500.00

Your program must define and call a method:
public static double secondsToJiffies(double userSeconds)
*/
'''
def secondsToJiffies(userSeconds):
    userSeconds = userSeconds * 100
    print("{:0.2f}".format(userSeconds))

try:
    secondsToJiffies(float(input()))
except ValueError:
    print("\n\n***Check Syntax***\nProgram will only takes numbers.")

File: Java-164/chapter.4/lab4.15/LabProgram.java
/*
A “jiffy” is the scientific name for 1/100th of a second. Given an input number of seconds, output the number of "jiffies."

Output each floating-point value with two digits after the decimal point, which can be achieved as follows:
System.out.printf("%.2f", yourValue);

Ex: If the input is:

15

the output is:

1500.00

Your program must define and call a method:
public static double secondsToJiffies(double userSeconds)
*/
import java.util.Scanner;

public class LabProgram {
   
   public static double secondsToJiffies(double userSeconds) {
       userSeconds = userSeconds * 100;
       return (double)userSeconds;
   }
   
   public static void main(String[] args) {
       Scanner scnr = new Scanner(System.in);
       double user_input = scnr.nextDouble();
      System.out.printf("%.2f", secondsToJiffies(user_input));
      scnr.close();

   }
}